src/__internal__/focus-trap/focus-trap.spec.js

```diff
@@ -9,15 +9,15 @@ import { ModalContext } from "../../components/modal/modal.component";
 jest.useFakeTimers();
 
 // eslint-disable-next-line
-const MockComponent = ({ children, triggerRefocusFlag, ...rest }) => {
+const MockComponent = ({ children, triggerRefocusFlag, tabIndex, ...rest }) => {
   const ref = useRef();
   const [isDisabled, setIsDisabled] = useState(false);
   return (
     <ModalContext.Provider
       value={{ isAnimationComplete: true, triggerRefocusFlag }}
```

Comment:
```suggestion
      const MockComponent = ({ children, triggerRefocusFlag, isAnimationComplete, tabIndex, ...rest }) => {
    ...
      return (
         <ModalContext.Provider
             value={{ isAnimationComplete, triggerRefocusFlag }}
   ...
```
`isAnimationComplete` is now true by default, I think we should test it by using the default value and maybe add a test where we set it to false and assert no focus etc (it seems we've missed writing a test for that previously 🤦 )